tests/test_smoke.py

```diff
@@ -2911,19 +2909,24 @@ def _get_service_name() -> str:
     '     sleep 10;'
     'done)')
 
+_SERVE_ENDPOINT_WAIT = (
+    'endpoint=$(sky serve status --endpoint {name}); '
+    'until ! echo "$endpoint" | grep "Controller"; '
```

Comment:
```suggestion
    'until ! echo "$endpoint" | grep "Controller is initializing"; '
```
more complete message for better readability?

Reply:
Sounds good! Updated!